src/components/markdown-renderer/hooks/use-trimmed-content.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { useSelector } from 'react-redux'
+import { ApplicationState } from '../../../redux'
+import { useMemo } from 'react'
+
+export const useTrimmedContent = (content: string): [trimmedContent: string, isTrimmed: boolean] => {
+  const maxLength = useSelector((state: ApplicationState) => state.config.maxDocumentLength)
+  const isTrimmed = content.length > maxLength
```

Comment:
As stated above, I'd rename this variable.

Reply:
same